df.buildings.xml

```diff
@@ -589,7 +573,7 @@
         <enum-item/> Invalid Type
         <enum-item name="MeadHall"/>
         <enum-item name="ThroneRoom"/>
-        <enum-item name="ActivityZone"/>
+        <enum-item/> formerly ActivityZone
```

Comment:
is this valid syntax?

Reply:
I guess so -- I see it below as well for Invalid Type (under DiningHall)

Reply:
yes, text nodes are treated as comments by codegen